src/cc/BPF.h

```diff
@@ -38,12 +39,15 @@ struct open_probe_t {
 };
 
 class USDT;
+class TableStorage;
 
 class BPF {
 public:
   static const int BPF_MAX_STACK_DEPTH = 127;
 
-  explicit BPF(unsigned int flag = 0) : bpf_module_(new BPFModule(flag)) {}
+  explicit BPF(unsigned int flag = 0, TableStorage *ts = nullptr)
+      : bpf_module_(new BPFModule(flag, ts)), ts_(ts)
```

Comment:
Why do we even need to keep `ts_` in the C++ class? Can we always go to something like `bpf_module->get_table_storage()` if we want to access the table storage? Especially given the `BPFModule` holds `unique_ptr` of the table storage class......